integration-cli/docker_cli_swarm_test.go

```diff
@@ -1079,10 +1079,9 @@ func (s *DockerSwarmSuite) TestSwarmLockUnlockCluster(c *check.C) {
 	c.Assert(getNodeStatus(c, d2), checker.Equals, swarm.LocalNodeStateActive)
 
 	// once it's caught up, d2 is set to not be locked
-	c.Assert(d2.Restart(), checker.IsNil)
```

Comment:
@aaronlehmann Ok, will remove the waiting for the state to not be in pending, then.